client/src/components/displayHomepage.js

```diff
@@ -0,0 +1,103 @@
+    FormControl
+} from "react-bootstrap";
+class RenderHomepagePictures extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            img: [],
+            name: []
+        };
+    }
+    loadImages() {
+        //Get all users that current user is following.
+        axios
+            .get("http://localhost:8000/api/follow/", {
+                headers: { token: sessionStorage.getItem("jwt") }
+            })
+            .then(res => {
+                this.setState({
+                    name: res.data.following
+                });
+                return res.data.following;
+            })
+            .then(result => {
+                //Get the photo file for each user
+                for (var i = 0; i < result.length; i++) {
+                    //Define current user.
+                    let user = {
+                        username: result[i]
+                    };
+                    //Create a promise for the files request.
+                    axios
+                        .post("http://localhost:8000/searchuser/files", user, {
+                            headers: { token: sessionStorage.getItem("jwt") }
+                        })
+                        .then(res => {
+                            //Push new information to this.state.img
+                            let currentarray = [...this.state.img];
+                            currentarray.push({
+                                image: res.data[res.data.length - 1].filename,
+                                description: res.data[res.data.length - 1].description
+                            });
+                            //Set the state to the new array
+                            this.setState({
+                                img: currentarray
+                            });
+                        });
+                }
+            });
+    }
+    componentDidMount() {
+        this.loadImages();
+    }
+    render() {
+        return (
+            <Container>
+                <Row>
+                    {this.state.img.map(item => {
+                        return (
+                            <Col>
+                                <Card
+                                    key={item}
+                                    style={{ width: "18rem" }}
+                                    bg="light"
+                                    border="dark"
+                                >
+                                    <Card.Img
+                                        variant="top"
+                                        src={"http://localhost:8000/image/" + item.image}
+                                        alt="Image could not be loaded"
+                                    />
+                                    <Card.Body>
+                                        <Card.Title>{item.description}</Card.Title>
+                                        <Card.Text>This will be the comment section</Card.Text>
```

Comment:
These sections will be updated with the comments section that I am still in the process of completing.